src/nidigital/system_tests/test_system_nidigital.py

```diff
@@ -251,6 +263,13 @@ def test_source_waveform_parallel_site_unique(multi_instrument_session, source_w
 
 
 def test_fetch_capture_waveform(multi_instrument_session):
+    '''Test methods for generating and capturing waveform to multiple sites in parallel.
```

Comment:
Test methods for using capture waveform with parallel acquisition.

Reply:
Updated.